clmm/theory/ccl.py

```diff
@@ -21,6 +21,17 @@
 
 __all__ = ['CCLCLMModeling', 'Modeling', 'Cosmology']+func_layer.__all__
 
+# Check which versions of ccl are currently supported
+__ccl_vmin = '2.6.2dev7'
```

Comment:
With the release of 2.7.0, `'2.6.2dev7'` becomes `'2.7.1.dev1'`. This will make CLMM not support any release versions.

Reply:
The number written there was actually a placeholder, I was going to ask you the correct versions to fill vmin/vmax.
I moved that to ` clmm/theory/_ccl_supported_versions.py` now btw